### PR TITLE
update snappy to 4.0.1

### PR DESCRIPTION
**Pull request by dudeche**
Hello @chaosgame, @nicks, @kylehg, 

Please review the following commits I made in branch 'ddb-update-snappy-version'.

8a8d628f33e628c4c3f83a334c6209d6e626fe9f (2015-09-21 13:21:45 -0700)
update snappy to 4.0.1

R=@chaosgame
R=@nicks
R=@kylehg

### CODE REVIEW

**Comment by nicks** [comment]
you have to run the install script. updating package.json does nothing on its own.

**Comment by nicks** [comment]
oh, ignore me, i thought this was medium2

lgtm!

**Comment by dudeche** [comment]
np - before I merge, how do I get an upped version onto npmjs.org?

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "hiredis": "0.4.1",
     "metrics": "0.1.6",
     "hashring": "3.2.0",
-    "snappy": "4.0.0"
```
Comment (chaosgame) [inline package.json]: Please also bump the package version so we can publish an update to npm

**Comment by dudeche** [comment]
updated PTAL

**Comment by chaosgame** [comment]
LGTM.

**Pull request merged**